### ISSUE TITLE
Testing framework

### ISSUE THREAD

**Issue by mxdi9i7** [NervJS/nerv#53]
Hi there, 

I've just started playing around with Nerv and I wanted to implement Jest as the testing framework but I'm running into some issues: The webpack dev server running into a compilation error.

My package.json looks like this: 
```
"scripts": {
    "dev": "webpack-dev-server --config webpack.config.js --mode development --port 8888",
    "test": "jest",
    "test:watch": "jest --watch"
  },
  "dependencies": {
    "nervjs": "^1.2.15"
  },
  "devDependencies": {
    "@babel/core": "^7.0.0-beta.42",
    "@babel/plugin-transform-react-jsx": "^7.0.0-beta.42",
    "@babel/preset-env": "^7.0.0-beta.42",
    "babel-core": "^6.26.0",
    "babel-jest": "^22.4.3",
    "babel-loader": "8.0.0-beta.0",
    "babel-plugin-transform-react-jsx": "^6.24.1",
    "babel-preset-env": "^1.6.1",
    "babel-preset-react": "^6.24.1",
    "html-webpack-plugin": "^3.0.7",
    "jest": "^22.4.3",
    "nerv-devtools": "^1.2.15",
    "react-test-renderer": "^16.2.0",
    "regenerator-runtime": "^0.11.1",
    "webpack": "^4.2.0",
    "webpack-cli": "^2.0.12",
    "webpack-dev-server": "^3.1.1"
  }
```

My babelrc is: 
```
{
  "presets": [
    [
      "env",
      {
        "spec": true,
        "useBuiltIns": false
      }
    ]
  ],
  "plugins": [
    [
      "transform-react-jsx",
      {
        "pragma": "Nerv.createElement"
      }
    ]
  ]
}
```

And my webpack.config.js is:
```
const path = require('path')
const HtmlWebpackPlugin = require('html-webpack-plugin')

module.exports = {
  entry: './src/index.js',
  output: {
    path: path.resolve(__dirname, 'dist'),
    filename: 'bundle.js'
  },
  module: {
    rules: [
      { test: /\.js$/, use: 'babel-loader' }
    ]
  },
  plugins: [
    new HtmlWebpackPlugin({
      filename: 'index.html',
      template: './index.html'
    })
  ]
}
```

After I finished the getting started guide on Jest, by installing a couple of dependencies, `yarn add --dev babel-jest babel-core regenerator-runtime`, the webpack dev server started to error: `ERROR in ./src/App.js
Module build failed: TypeError: Cannot read property 'bindings' of null`

![image](https://user-images.githubusercontent.com/13799221/37738840-a0f591a6-2d2e-11e8-9021-891e95a7cb2d.png)

**Comment by yuche** [comment]
This is issue is possibly related to https://github.com/babel/babel/issues/6186 .

**Comment by mxdi9i7** [comment]
Resolved with babel 6.2.0:

```
"devDependencies": {
    "babel-core": "^6.26.0",
    "babel-jest": "^22.4.3",
    "babel-loader": "^7.1.4",
    "babel-plugin-transform-react-jsx": "^6.24.1",
    "babel-preset-react": "^6.24.1",
    "css-loader": "^0.28.11",
    "extract-text-webpack-plugin": "next",
    "html-webpack-plugin": "^3.0.7",
    "jest": "^22.4.3",
    "nerv-devtools": "^1.2.15",
    "node-sass": "^4.7.2",
    "react-test-renderer": "^16.2.0",
    "regenerator-runtime": "^0.11.1",
    "sass-loader": "^6.0.7",
    "style-loader": "^0.20.3",
    "webpack": "^4.2.0",
    "webpack-cli": "^2.0.12",
    "webpack-dev-server": "^3.1.1"
  }
```

Thanks for the help
